runtime/tutorials/set_up_environment.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Setting up your Environment"
+---
+
+Deno is a secure runtime for JavaScript and TypeScript that comes with a set of
+built-in tools and utilities. In this tutorial, we’ll walk you through setting
+up your development environment for Deno using Visual Studio Code (VSCode).
```

Comment:
```suggestion
up your development environment for Deno using Visual Studio Code (VS Code).
```

Very minor stuff... I've heard that VS Code is official abbreviation. I can't find clear source, but it looks like their official FAQ consistently uses VS Code https://code.visualstudio.com/docs/supporting/faq

Reply:
Ooh, nice catch!